AppKit/CPRuleEditor/CPPredicateEditorRowTemplate.j

```diff
@@ -474,6 +474,8 @@ CPTransformableAttributeType = 1800;
         value = [aView intValue];
     else if (attributeType == CPBooleanAttributeType)
         value = [aView state];
+    else if (attributeType == CPDateAttributeType)
+        value = [aView dateValue];
     else
         value = [aView stringValue];
 
```

Comment:
We should be able to just return the control objectValue. It works for the date (datePicker objectValue), boolean and string attributes. It should work for the integer and float attributes if the control has a number formatter ...